public/app/features/query/components/QueryGroup.tsx

```diff
@@ -477,34 +478,34 @@ export class QueryGroup extends PureComponent<Props, State> {
   }
 }
 
-const getStyles = stylesFactory(() => {
-  const { theme } = config;
+export const QueryGroup = withTheme2(UnThemedQueryGroup);
 
+function getStyles(theme: GrafanaTheme2, isDragActive?: boolean) {
```

Comment:
don't see isDragActive being used here

Reply:
Ah, a remnant of past UX iterations. Will remove.